### ISSUE TITLE
User cannot access multi-select options with a keyboard

### ISSUE THREAD

**Issue by dakahn** [carbon-design-system/carbon#1472]
## Environment
Chrome 70
macOS Mojave

## Violation
The user with a keyboard cannot access the options in the multi-select and inline-multi-select drop downs

**Comment by asudoh** [comment]
Thanks @dakahn for reporting! As we talked offline, multi select is a React-only component as of today, and the one we have in our dev env is for demonstrating markup/styles only.
